onnx/defs/tensor/defs.cc

```diff
@@ -1185,4 +1185,122 @@ ONNX_OPERATOR_SET_SCHEMA(
             "T1",
             {"tensor(bool)"},
             "Constrains to boolean tensors."));
+
+static const char* OneHot_ver9_doc = R"DOC(
+    Produces a one-hot tensor based on inputs.
+    The locations represented by the index values in the 'indices' input tensor will have 'on_value' 
+    and the other locations will have 'off_value' in the output tensor, where 'on_value' and 'off_value' 
+    are specified as part of required input argument 'values', which is a two-element tensor of format  
+    [off_value, on_value]. The rank of the output tensor will be one greater than the rank of the 
+    input tensor. The additional dimension is for one-hot representation. The additional dimension will 
+    be inserted at the position specified by 'axis'. If 'axis' is not specified then then additional 
+    dimension will be inserted as the innermost dimension, i.e. axis=-1. The size of the additional 
+    dimension is specified by required scalar input 'depth'. The type of the output tensor is the same 
+    as the type of the 'values' input. Any entries in the 'indices' input tensor with values outside 
+    the range [0, depth) will result in one-hot representation with all 'off_value' values in the 
+    output tensor.
+)DOC";
+
+ONNX_OPERATOR_SET_SCHEMA(
+    OneHot,
+    9,
+    OpSchema()
+        .SetDoc(OneHot_ver9_doc)
+        .Attr(
+            "axis",
+            "(Optional) Axis along which one-hot representation in added. Default: axis=-1.",
```

Comment:
Let's mention axis=-1 means the last dimension.

Reply:
Done.